subgraphs/network/schema.graphql

```diff
@@ -0,0 +1,325 @@
+  dailyMeanGasUsed: BigDecimal!
+
+  " Mean of gas limit per block during this day "
+  dailyMeanGasLimit: BigDecimal!
+
+  " Total amount of fees burned "
+  cumulativeBurntFees: BigInt!
+
+  " Amount of burnt fees for this day "
+  dailyBurntFees: BigInt!
+
+  " Total rewards emitted up until this day "
+  cumulativeRewards: BigInt
+
+  " Rewards emitted for this day (ie, new supply this day) "
+  dailyRewards: BigInt
+
+  " First timestamp of this entity (used to calculate averages) "
+  firstTimestamp: BigInt!
+
+  " Daily mean block interval time in seconds during this hour "
+  dailyMeanBlockInterval: BigDecimal!
+
+  " Cumulative size of all blocks created in this day "
+  dailyCumulativeSize: BigInt!
+
+  " Daily mean size in bytes of a block during this day "
+  dailyMeanBlockSize: BigDecimal!
+
+  " All of the blocks created during this day "
+  blocks: [Block!]! @derivedFrom(field: "day")
+}
+
+type HourlySnapshot @entity {
+  " { number of hours since UNIX epoch time } "
+  id: ID!
+
+  " ID of the network this snapshot belongs to "
+  network: Network!
+
+  " Number of cumulative ununique authors "
+  cumulativeUniqueAuthors: Int!
+
+  " Number of unique hourly active authors "
+  hourlyActiveAuthors: Int!
+
+  " Block height at this hour (ie, blockNumber) "
+  blockHeight: Int!
+
+  " Timestamp of this snapshot "
+  timestamp: BigInt!
+
+  " Blocks created during this hour "
+  hourlyBlocks: Int!
+
+  " Blocks/day calculated at the last block this hour "
+  blocksPerDay: BigDecimal!
+
+  " Total difficulty up until this hour "
+  cumulativeDifficulty: BigInt!
+
+  " Difficulty during this hour "
+  hourlyDifficulty: BigInt!
+
+  " Average difficulty for this hour "
+  hourlyMeanDifficulty: BigDecimal!
+
+  " Sum of all gas used during this hour "
+  hourlyCumulativeGasUsed: BigInt!
+
+  " Sum of all gas limit during this hour "
+  hourlyCumulativeGasLimit: BigInt!
+
+  " Mean of gas used per block during this hour "
+  hourlyMeanGasUsed: BigDecimal!
+
+  " Mean of gas limit per block during this hour "
+  hourlyMeanGasLimit: BigDecimal!
+
+  " Total amount of fees burned "
+  cumulativeBurntFees: BigInt!
+
+  " Amount of burnt fees for this hour "
+  hourlyBurntFees: BigInt!
+
+  " Total rewards emitted up until this hour "
+  cumulativeRewards: BigInt
+
+  " Rewards emitted for this hour (ie, new supply this hour) "
+  hourlyRewards: BigInt
+
+  " First timestamp of this entity (used to calculate averages) "
+  firstTimestamp: BigInt!
+
+  " Hourly mean block interval time in seconds during this hour "
+  hourlyMeanBlockInterval: BigDecimal!
+
+  " Cumulative size of all blocks created in this hour "
+  hourlyCumulativeSize: BigInt!
+
+  " Hourly mean size of a block during this hour "
+  hourlyMeanBlockSize: BigDecimal!
+
+  " All of the blocks created during this day "
+  blocks: [Block!]! @derivedFrom(field: "hour")
+}
+
+##########################
+#### Block-level Data ####
+##########################
+
+type Block @entity {
```

Comment:
are we able to get transactions per block?

Reply:
not on evm or near, will check on the other two

Reply:
hmm, okay. That's unfortunate :(